transform-plugins/src/main/java/io/cdap/plugin/ValueMapper.java

```diff
@@ -82,6 +85,15 @@ public Config(String mapping, String defaults) {
     }
   }
 
+  @Override
+  public void prepareRun(StageSubmitterContext context) throws Exception {
+    super.prepareRun(context);
+    context.record(
```

Comment:
take a look at how the output schema is generated from the inputSchema, https://github.com/cdapio/hydrator-plugins/blob/develop/transform-plugins/src/main/java/io/cdap/plugin/ValueMapper.java#L130, if it is not in the mapping list, it is identity, if it is, it gets mapped to some other value.